model/flow/External.yaml

```diff
@@ -0,0 +1,3 @@
+model:
+  name: User
+  behavior:
```

Comment:
This is technically valid, but it looks weird to define an empty behavior for a model with no inputs/outputs.

Reply:
I think what we'd actually want to do is define it like this (i.e. an empty list of behaviors):

```yaml
model:
  name: User
  behavior: []
```

Reply:
```suggestion
  behavior: []
```

Reply:
Why is it valid?  Behavior has required fields of name and type.  Given these fields aren't present shouldn't it be invalid?